packages/office-ui-fabric-react/src/components/MarqueeSelection/__snapshots__/MarqueeSelection.test.tsx.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MarqueeSelection renders MarqueeSelection correctly 1`] = `
+<div
+  className=
+
+      {
+        cursor: default;
+        position: relative;
+      }
```

Comment:
Hmm, can you try using mount instead? It should snapshot more of the styling.. however, in cases where a bunch of other components are used it can cause huge outputs. In this case, however, I think it will work.

Reply:
Actually sorry, the problem here could also be that this is the only render content until some other event occurs, which you may want to consider simulating just to verify styling...